backend/fms_core/models/transferred_sample.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+from .sample import Sample
+from .protocol import Protocol
+from .process import Process
+from .process_sample import ProcessSample
+
+
+__all__ = ["TransferredSample"]
+
+
+class TransferredSampleManager(models.Manager):
```

Comment:
Not sure we need this model ... displaying the transfered samples on the backend do not appear really useful and if we keep doing this we'll end up having dozens of proxy models. Maybe we need to decide if we want to display all this in the backend.

Reply:
I need to register TransferredSample model at some places. So should I just remove the TransferredSampleManager class, and fetch the objects directly in TransferredSample?

Reply:
No i meant we do not need both ... transferredSample is not a real model it is just a stub for admin to display a list of transfered samples, but i do not think we should do that for each process. We will not be going back to this interface and it has no use for admin since there is (or will be ;) ) an API endpoint to list those transfers and extraction ...

Reply:
Let's keep it for now. The back-end allows double-checking the data entered by displaying it in a convenient table-format before saving it. It is a feature that is requested on the front-end, but for now it's not there.